adversarial_text/graphs.py

```diff
@@ -81,7 +81,9 @@
 # Regularization
 flags.DEFINE_float('max_grad_norm', 1.0,
                    'Clip the global gradient norm to this value.')
-flags.DEFINE_float('keep_prob_emb', 1.0, 'keep probability on embedding layer')
+flags.DEFINE_float('keep_prob_emb', 0.5, 
```

Comment:
Can you leave the default at 1.0 but keep the docstring saying optimal is 0.5?

And add period and space after 'layer'